crates/ty_python_semantic/resources/mdtest/dataclasses/fields.md

```diff
@@ -37,7 +37,7 @@ class Data:
     content: list[int] = field(default_factory=list)
     timestamp: datetime = field(default_factory=datetime.now, init=False)
 
-# revealed: (self: Data, content: list[int] = list[Unknown]) -> None
+# revealed: (self: Data, content: list[int] = list[int]) -> None
```

Comment:
This shows that our previous approach, calling `default_factory` manually, was apparently too naive.